wiki/Game_Modes/osu!/pt-br.md

```diff
@@ -1,10 +1,10 @@
 <!-- wiki -->
-[Game_Modes wikilink]: ../ "Modalides de Jogo"
-[Play_Styles#osu! wikilink]: /wiki/Play_Styles/ "mais informações podem ser encontradas podem ser encontradas sob osu! Estilos de Jogo"
-[Score#osu!SV wikilink]: /wiki/Score/#osu "mais informações podem ser encontradas em osu! Valores de Pontuação"
-[Options#Keyboard wikilink]: /wiki/Options/ "mais informações podem ser encontradas em 'Opções', sob 'Teclado'"
-[Skinning#osu! wikilink]: /wiki/Skinning/osu!/ "osu! Skinning"
-[Mascots#pippi wikilink]: /wiki/Mascots/#pippi "mais informações podem ser encontradas em pippi"
+[Game_Modes wikilink]: .. "Modalides de Jogo"
```

Comment:
dunno if this works as intended, but i assume its okay

Reply:
hmm... maybe I should go back and replace all `..` anyway, you pointed out some issue on the website with this a while ago right

Reply:
i don't remember any `../` issue 🤔 

https://osu.ppy.sh/help/wiki/Game_Modes/osu!?locale=pt-br

first link works fine